web/index.html

```diff
@@ -421,14 +527,49 @@ <h3>Used in</h3>
                     }
                     return n.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')
                 },
-                loadChunk: function (chunk) {
-                    fetch(DATA_PREFIX + `chunk-${chunk}.json`)
-                        .then(response => { console.log("Loaded chunk", chunk); return response.json() })
+                loadChunk: async function (chunk) {
+                    return fetch(DATA_PREFIX + `chunk-${chunk}.json`)
+                        .then(response => {
+                            console.log("Loaded chunk", chunk);
+                            return response.json()
+                        })
                         .then(data => {
                             this.data.data = Object.freeze(Object.assign({}, this.data.data, data))
                             this.data.metadata.loadedChunks.push(chunk)
-                            console.log(this.data.data)
-                        })
+                        });
+                },
+                calcCraftingTree: async function (element) {
```

Comment:
This can be a computed property

Reply:
It really shouldn't be. Based on the way async callbacks work, you shouldn't every try to use them as computed values -- even if you can get vue to stop yelling at you that isn't ever going to work as intended. This should obviously be an async function, because it loads more data whenever it needs to load additional chunks, and loading chunks async in definitely the prefered method as it makes the site feel so much smoother and is even when multiple chunks are loaded it is only a fraction of a second delay.

Reply:
TODO add crafting tree to data chunks